vocabs/v1/context.jsonld

```diff
@@ -1,89 +1,59 @@
 {
   "@context": {
-    "@version": 1.1,
+    "@protected": true,
     "id": "@id",
     "type": "@type",
-    "dc": "http://purl.org/dc/terms/",
-    "schema": "http://schema.org/",
-    "sec": "https://w3id.org/security#",
-    "didns": "https://www.w3.org/ns/did#",
-    "xsd": "http://www.w3.org/2001/XMLSchema#",
-    "EcdsaSecp256k1VerificationKey2019": "sec:EcdsaSecp256k1VerificationKey2019",
-    "Ed25519Signature2018": "sec:Ed25519Signature2018",
-    "Ed25519VerificationKey2018": "sec:Ed25519VerificationKey2018",
-    "JsonWebKey2020": "sec:JsonWebKey2020",
-    "JsonWebSignature2020": "sec:JsonWebSignature2020",
-    "Bls12381G1Key2020": "sec:Bls12381G1Key2020",
-    "Bls12381G2Key2020": "sec:Bls12381G2Key2020",
-    "RsaVerificationKey2018": "sec:RsaVerificationKey2018",
-    "SchnorrSecp256k1VerificationKey2019": "sec:SchnorrSecp256k1VerificationKey2019",
-    "X25519KeyAgreementKey2019": "sec:X25519KeyAgreementKey2019",
-    "ServiceEndpointProxyService": "didns:ServiceEndpointProxyService",
-    "LinkedDomains": "https://identity.foundation/.well-known/resources/did-configuration/#LinkedDomains",
+
     "alsoKnownAs": {
       "@id": "https://www.w3.org/ns/activitystreams#alsoKnownAs",
       "@type": "@id",
       "@container": "@set"
     },
     "assertionMethod": {
-      "@id": "sec:assertionMethod",
+      "@id": "https://w3id.org/security#assertionMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "authentication": {
-      "@id": "sec:authenticationMethod",
+      "@id": "https://w3id.org/security#authenticationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityDelegation": {
-      "@id": "sec:capabilityDelegationMethod",
+      "@id": "https://w3id.org/security#capabilityDelegationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityInvocation": {
-      "@id": "sec:capabilityInvocationMethod",
+      "@id": "https://w3id.org/security#capabilityInvocationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "controller": {
-      "@id": "sec:controller",
+      "@id": "https://w3id.org/security#controller",
       "@type": "@id"
     },
-    "created": {
-      "@id": "dc:created",
-      "@type": "xsd:dateTime"
-    },
-    "blockchainAccountId": "sec:blockchainAccountId",
     "keyAgreement": {
-      "@id": "sec:keyAgreementMethod",
+      "@id": "https://w3id.org/security#keyAgreementMethod",
       "@type": "@id",
       "@container": "@set"
     },
-    "publicKey": {
```

Comment:
publicKey not being retained for backwards compatibility?

Reply:
No, publicKey has been deprecated for a while now. It's usually a bad practice to keep deprecated things around, especially since it would lead to two code paths in crypto libraries (and we really want to try and reduce things to a few code paths as possible to reduce attack surface).